Assignment-12/src/classes/ArchiveConfigmdt_HandlerTest.cls

```diff
@@ -0,0 +1,46 @@
+/** 
+ * Purpose          :   Test class of ArchiveConfigmdt_Handler class
+ * 
+ * Created by       :   Sajal Mittal
+ * 
+ * Created Date     :   02-01-2025
+ * 
+ * Rivision Logs    :   V_1.0 - Created
+ * 
+**/
+@isTest(seeAllData = TRUE)
+private class ArchiveConfigmdt_HandlerTest {    
+
+    // Test method to test scheduler class ArchiveConfigmdt_Handler
+    @isTest
+    static void testArchiveConfigmdtHandler() {  
+        
+        Test.startTest();
+        String cronExp = '0 0 0 * * ? 2042';
```

Comment:
One tab (Four Space) gap should be present in between Test.startTest() and Test.stopTest().